src/components/Navbar.jsx

```diff
@@ -6,7 +6,7 @@ import { Plus } from "lucide-react";
 import Avatar from "../assets/cat.png";
 
 const Navbar = () => {
-  const user = "jessjelly"
+  const user = "jessjelly";
```

Comment:
Oooh it could be cool to add in some of what David taught this morning about context here!